CONTRIBUTING.md

```diff
@@ -86,3 +86,54 @@ described below:
   "proposal finalized" label.  This indicates that it is ready to be
   implemented by a platform developer, see the [process](#contributing) above.
 - Once a Pull Request is approved, it will be merged into the 'master' branch.
+
+## Release Process
+
+Any member of the PMC can request that the a SHA on master (the **Release SHA**)
+is ready to be released into a new version of the spec. They will do this by
+creating a new PR with the title of the proposed release. For example,
+**"Release Proposal: v$major.$minor"**.
+
+### Prepare a PR
+
+1. In a fork, create a new branch called "v$major.$minor-rc" from the **Release SHA**.
+2. Create a new commit (the **Release Commit**) with the following changes:
+  * Update [release-notes.md](https://github.com/openservicebrokerapi/servicebroker/blob/master/release-notes.md) detailing the changes that are to be released in this version.
+  * Update [README.md](https://github.com/openservicebrokerapi/servicebroker/blob/master/README.md) with an updated _Latest Release_ subheading and link to the latest released specification.
+  * Update [spec.md](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md) with an updated _Changes Since v..._ section (and link from table of contents) containing a copy of the relevant release notes, and with any references to the previous version of the specification (i.e. the `X-Broker-Api-Version` headers) updated.
```

Comment:
The references from the 'Changes since ....' section to older releases seem like they are removed in our process, not just updated.  The release notes holds the permanent list of changes, right?